<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>176</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ramp parameters</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0,0">
   <item row="2" column="2">
    <widget class="QLabel" name="label_unit_t0">
     <property name="text">
      <string>s</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="label_unit0">
     <property name="text">
      <string>Unit</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_t0">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="LineEdit" name="lineedit_v0"/>
   </item>
   <item row="2" column="1">
    <widget class="LineEdit" name="lineedit_t0"/>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_value">
     <property name="text">
      <string>Value</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="LineEdit" name="lineedit_t1"/>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_time">
     <property name="text">
      <string>Time</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_reset">
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_ok">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="4">
    <widget class="QLabel" name="label_unit1">
     <property name="text">
      <string>Unit</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="5">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Define ramp for </string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_t1">
     <property name="text">
      <string>End</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="LineEdit" name="lineedit_v1"/>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label_unit_t1">
     <property name="text">
      <string>s</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">mfixgui.widgets.base</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineedit_t0</tabstop>
  <tabstop>lineedit_v0</tabstop>
  <tabstop>lineedit_t1</tabstop>
  <tabstop>lineedit_v1</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
